docs/modelserving/v1beta1/llm/huggingface/README.md

```diff
@@ -4,8 +4,11 @@ The Hugging Face LLM serving runtime implements a runtime that can serve Hugging
 In this example, we deploy a Llama2 model from Hugging Face by running an `InferenceService` with [Hugging Face Serving runtime](https://github.com/kserve/kserve/tree/master/python/huggingfaceserver). Based on the performance requirement for large language models, KServe chooses to perform the inference using a more optimized inference engine like [vLLM](https://github.com/vllm-project/vllm) for text generation models.
 
 ### Serve the Hugging Face LLM model using vLLM
-KServe Hugging Face runtime by default uses vLLM to serve the LLM models for faster inference, higher throughput than Hugging Face API, implemented with paged attention, continous batching, optmized CUDA kernel. 
-You can still use `--backend=huggingface` in the container args to fall back to perform the inference using Hugging Face API.
+
+KServe Hugging Face runtime by default uses vLLM to serve the LLM models for faster inference and higher throughput than the Hugging Face API, implemented with paged attention, continuous batching and an optimized CUDA kernel. 
+
+You can still use `--disable_vllm` flag to fall back to perform the inference using Hugging Face API.
```

Comment:
This flag is no longer available. We should use --backend=huggingface

Reply:
ok thanks i think this got messed up in the rebase